src/Data/Unrestricted/Internal/Instances.hs

```diff
@@ -426,24 +313,36 @@ instance Prelude.Traversable Ur where
 
 -- Some stock instances
 deriving instance Consumable a => Consumable (Sum a)
+
```

Comment:
Ah, but I see some such lines have been added. Is that an Ormolu thing?

Reply:
As you guessed it, it is an ormolu thing. Maybe I could rewrite it in the "packed" style (so as to make the diff cleaner), as PR https://github.com/tweag/linear-base/pull/355 will ormolize the whole codebase later?

Reply:
It would have been preferable. But it's probably not worth the time investment now.